File: sqlite3/python/update/sqlite3_update.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	update/sqlite_update.py
#
#					Sep/06/2016
#
# --------------------------------------------------------
import	sys
import	sqlite3
import	string
#
sys.path.append ('/var/www/data_base/common/python_common')
from sql_manipulate import sql_update_proc
#
# --------------------------------------------------------
sys.stderr.write ("*** 開始 ***\n")
#
file_in = sys.argv[1]
key_in = sys.argv[2]
population_in = int (sys.argv[3])
print ("%s\t%d" % (key_in, population_in))
#
conn = sqlite3.connect (file_in)
conn.text_factory=str
#
cursor = conn.cursor ()
#
sql_update_proc	(cursor,key_in,population_in)
#
conn.commit ()
#
cursor.close ()
conn.close ()
#
sys.stderr.write ("*** 終了 ***\n")
#
# --------------------------------------------------------

File: sqlite3/php/update/sqlite3_update.php
#! /usr/bin/php
<?php
/*
	sqlite3_update.php

					Jul/25/2018

*/
// --------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);

include "sql_manipulate.php";

// --------------------------------------------------------------------
fputs (STDERR,"*** 開始 ***\n");
$file_in = $argv[1];
$id_in = $argv[2];
$population_in = $argv[3];

print $file_in . "\t" . $id_in . "\t" . $population_in . "\n";

$dbcon = new PDO ("sqlite:" . $file_in);

sql_update_proc ($dbcon,$id_in,$population_in);

$dbcon = null;

fputs (STDERR,"*** 終了 ***\n");
// --------------------------------------------------------------------
?>	
